### ISSUE TITLE
Runtime Error: generator didn't stop after throw()

### ISSUE THREAD

**Issue by metanomy** [sci-wms/sci-wms#83]
TL;DR: is CF standard_name and/or units required in variable definition to display as layer?
1. Enabled fvcom_mesh variable

![image](https://cloud.githubusercontent.com/assets/11446338/10323467/3ce1ce64-6c51-11e5-9efb-83c75b9257a1.png)
1. Selected fvcom_mesh():
   ![image](https://cloud.githubusercontent.com/assets/11446338/10323447/176f866c-6c51-11e5-8301-288e96558548.png)

From logs:

ERROR:django.request:Internal Server Error: /wms/datasets/wfsfvcomagg
Traceback (most recent call last):
  File "/opt/conda/lib/python2.7/site-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, _callback_args, *_callback_kwargs)
  File "/opt/conda/lib/python2.7/site-packages/django/views/generic/base.py", line 69, in view
    return self.dispatch(request, _args, *_kwargs)
  File "/opt/conda/lib/python2.7/site-packages/django/views/generic/base.py", line 87, in dispatch
    return handler(request, _args, *_kwargs)
  File "/srv/sci-wms/wms/views.py", line 271, in get
    response = getattr(dataset, reqtype.lower())(layer, request)
  File "/srv/sci-wms/wms/models/datasets/ugrid.py", line 325, in getmap
    raise NotImplementedError('Image type "{}" is not supported.'.format(request.GET['image_type']))
  File "/opt/conda/lib/python2.7/contextlib.py", line 36, in **exit**
    raise RuntimeError("generator didn't stop after throw()")
RuntimeError: generator didn't stop after throw()
ERROR:tornado.access:500 GET /wms/datasets/wfsfvcomagg?service=WMS&request=GetMap&version=1.1.1&layers=fvcom_mesh&styles=filledcontours_jet&format=image/png&transparent=true&height=256&width=256&colorscalerange=0,1&srs=EPSG:3857&bbox=-9236039.001754416,3228700.0747658443,-9216471.12251341,3248267.954006849 (72.185.126.234) 2760.69ms

**Comment by rsignell-usgs** [comment]
@metanomy , sci-wms will only show data variables, and the `fvcom_mesh` is not a data variable.

**Comment by metanomy** [comment]
@rsignell-usgs Doh! Trying to see how to answer @geofishfl request to have a visual representation of the mesh of a given model. That way we could compare the mesh between models not just their data output.

**Comment by kwilcox** [comment]
@metanomy I like the idea of being able to show model grids as a wms layer. Can you start a new issue?
